Modules/+Drivers/@Wavemeter/Wavemeter.m

```diff
@@ -126,11 +126,17 @@
                 output = obj.com(cmd,obj.channel,0);
             end
             if output <= 0
-                if output == -1
-                    msg = 'Low signal';
+                if output == 0 || output == -5 || output == -6 || output == -15
+                    msg = 'Not Available';
+                elseif output == -1
+                    msg = 'No signal';
                 elseif output == -2
-                    msg = 'High signal';
+                    msg = 'Bad signal';
```

Comment:
What does bad signal mean? just like not coherent enough to read on the wavemeter?

Reply:
https://www.highfinesse.com/en/support/frequently-asked.html#question-3